blas/netlib/generate_blas.go

```diff
@@ -368,24 +369,13 @@ func gemmShape(buf *bytes.Buffer, d binding.Declaration, p binding.Parameter) bo
 		return false // Come back later.
 	}
 
-	fmt.Fprint(buf, `	var rowA, colA, rowB, colB int
-	if tA == C.CblasNoTrans {
-		rowA, colA = m, k
-	} else {
-		rowA, colA = k, m
-	}
-	if tB == C.CblasNoTrans {
-		rowB, colB = k, n
-	} else {
-		rowB, colB = n, k
-	}
-	if lda*(rowA-1)+colA > len(a) || lda < max(1, colA) {
+	fmt.Fprint(buf, `	if lda*(rowA-1)+colA > len(a) {
```

Comment:
Can you add a note that the row/col vars have already been declared in `leadingDim`.